<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Flippr</class>
 <widget class="QMainWindow" name="Flippr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>174</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Flippr</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="controls_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Controls</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="controls_layout">
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="comp_spin">
        <property name="suffix">
         <string/>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.001000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="decay_spin">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="value">
         <double>0.250000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="comp_label">
        <property name="text">
         <string>Compensation</string>
        </property>
        <property name="buddy">
         <cstring>amplitude_spin</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="on_button">
        <property name="text">
         <string>ON/OFF</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLineEdit" name="running_indicator">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="decay_label">
        <property name="text">
         <string>Constant</string>
        </property>
        <property name="buddy">
         <cstring>decay_spin</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="amplitude_label">
        <property name="text">
         <string>Amplitude</string>
        </property>
        <property name="buddy">
         <cstring>amplitude_spin</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="QDoubleSpinBox" name="amplitude_spin">
        <property name="suffix">
         <string/>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.001000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="diagnostics_box">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>180</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Diagnostics</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>160</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="diagnostics_layout">
      <item row="0" column="0">
       <widget class="QLabel" name="frequency_label">
        <property name="text">
         <string>Frequency</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="missed_lineedit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="freq_lineedit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="missed_label">
        <property name="text">
         <string>Missed</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="filename_label">
        <property name="text">
         <string>Filename</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="filename_lineedit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="pulseOutput" native="true">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>amplitude_spin</tabstop>
  <tabstop>decay_spin</tabstop>
  <tabstop>on_button</tabstop>
  <tabstop>running_indicator</tabstop>
  <tabstop>freq_lineedit</tabstop>
  <tabstop>missed_lineedit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
